### PR TITLE
Adding frontend feature

### PR DESCRIPTION
**Pull request by SoumyaSubhrajit**
Adding front-end code to the codebase

### CODE REVIEW

File: server/errors/errorHandler.js [hunk 1]
```diff
@@ -1,39 +1,62 @@
...
-          case 401:
-            errorMessage = 'Unauthorized'
-             break;
-           case 403:
-             errorMessage = 'BALA'
-           case 404:
-            errorMessage = 'Not Found'
-            case 429:
-            errorMessage = 'Too Many Requests'
-            case 500:
```
Comment (SoumyaSubhrajit) [inline server/errors/errorHandler.js:27]: Add more error handle edge cases and test them in JEST.

**Review by SoumyaSubhrajit — COMMENTED** [review]
Good to go
